lib/lost-center.js

```diff
@@ -32,6 +36,8 @@ module.exports = function lostCenterDecl(css, settings, result) {
       lostCenterFlexbox = 'no-flex';
     }
 
+    lostCenterFlexbox = lgLogic.parseLostProperty(decl.parent.nodes, 'lost-center-flexbox', lostCenterFlexbox);
+
```

Comment:
Did you mean to leave this line blank? Wasn't sure it it was significant? If it is I'll shut up, otherwise it would neaten it up a very tiny bit :)

Reply:
You're not the first person who's commented on my whitespace. I love it to a fault. I'll clean that up! Thanks for catching that!

- [ ] Add linting for whitespace